tests/testthat/test-template-content.R

```diff
@@ -1,41 +1,160 @@
 context("test-template-content")
 
+n_matches <- function(page, pattern) {
+  length(grep(pattern, page, fixed = TRUE, value = TRUE))
+}
+
 # Open Graph ------------------------------------------
 
-test_that("og tags are populated on home, reference, and articles", {
+describe("og tags are populated on home, reference, and articles", {
```

Comment:
Can you please stick with the testing style used in the rest of the package?

Reply:
Sure! I changed back to `test_that(...)` style.